src/components/Header/HeaderUser.jsx

```diff
@@ -0,0 +1,201 @@
+import React, { useState } from "react";
+import { useSelector, useDispatch } from "react-redux"
+import { NavLink, useLocation } from 'react-router-dom'
+import CyrillicToTranslit from 'cyrillic-to-translit-js'
```

Comment:
pls remove this library

Reply:
Oh yes, indeed, I completely missed the point that we now use the first letter of email. And there is no Cyrillic alphabet in the mail name))))) THANK YOU